_includes/docs_menu.html

```diff
@@ -2,22 +2,23 @@
   {% include search_form.html %}
   <ul class="nav nav-list">
     <li>Overview</li>
-    <li><a href="{{ site.baseurl }}/docs/">Get Started</a></li>
-    <li><a href="{{ site.baseurl }}/docs/example">Example</a></li>
+    <li><a href="{{ site.baseurl }}/docs/">Tableau Connector SDK</a></li>
+    <li><a href="{{ site.baseurl }}/docs/example">Connector Examples</a></li>
     <li>Basic Concepts</li>
     <li><a href="{{ site.baseurl }}/docs/design">Design Considerations</a></li>
-    <li><a href="{{ site.baseurl }}/docs/dialect">Create a TDD File</a></li>
-    <li><a href="{{ site.baseurl }}/docs/ui">Building the Connection Dialog</a></li>
-    <li><a href="{{ site.baseurl }}/docs/tdvt">Test Your Connector Using TDVT</a></li>
```

Comment:
Can we leave TDVT in the title somewhere? TDVT is probably one of the the things people are looking for documentation for the most.